communities/microgalaxy/lab/sections/3_microbiome.yml

```diff
@@ -1,9 +1,495 @@
+            button_tip: "Visit RefSeq"
+            description_md: >
+              A high-quality reference database of curated genomic sequences from NCBI.
+              Used in tools such as [<code>Kraken2</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/kraken2/kraken2) and [<code>BLAST</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastdbcmd_info)
+          - title_md: <code>SILVA</code> - 16S/18S/28S rRNA gene reference database
+            button_link: "https://www.arb-silva.de/"
+            button_tip: "Visit SILVA"
+            description_md: >
+              SILVA provides high-quality reference sequences for rRNA-based microbial classification.
+              Used in tools such as [<code>LotuS2</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/earlhaminst/lotus2/lotus2), [<code>dada2: assignTaxonomy and addSpecies</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/dada2_assigntaxonomyaddspecies/dada2_assignTaxonomyAddspecies), [<code>Kraken2</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/kraken2/kraken2), Mothur and QIIME
+          - title_md: <code>Greengenes</code> - 16S rRNA reference database
+            button_link: "https://greengenes.secondgenome.com/"
+            button_tip: "Visit Greengenes"
+            description_md: >
+              A widely used 16S rRNA gene reference for microbial classification.
+              Used in tools such as [<code>dada2: assignTaxonomy and addSpecies</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/dada2_assigntaxonomyaddspecies/dada2_assignTaxonomyAddspecies)
+          - title_md: <code>GTDB</code> - Genome Taxonomy Database
+            button_link: "https://gtdb.ecogenomic.org/"
+            button_tip: "Visit GTDB"
+            description_md: >
+              A curated genome-based taxonomy database for bacteria and archaea.
+              Used in tools such as [<code>GTDB-Tk</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/gtdbtk_classify_wf/gtdbtk_classify_wf) and [<code>dada2: assignTaxonomy and addSpecies</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/dada2_assigntaxonomyaddspecies/dada2_assignTaxonomyAddspecies)
+          - title_md: <code>ChocoPhlAn (MetaPhlAn)</code> - Marker gene database for metagenomic profiling
+            button_link: "https://bitbucket.org/biobakery/metaphlan/src/master/"
+            button_tip: "Visit MetaPhlAn"
+            description_md: >
+              ChocoPhlAn is the core marker gene database used by MetaPhlAn for microbial community profiling.
+              Used in tools such as [<code>MetaPhlAn</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/metaphlan/metaphlan)
+
+      - id: functional-annotation
+        title: Functional Annotation Databases
```

Comment:
You could add Gene Ontology there? and UniProt?
Can you check databases used in HUMAnN?

Reply:
HUMAnN I will added for Uniref and ChocoPhlAn